executor/runtime/docker/capabilities.go

```diff
@@ -83,6 +84,16 @@ func setupAdditionalCapabilities(c *runtimeTypes.Container, hostCfg *container.H
 			CgroupPermissions: "rmw",
 		})
 
+		value, exists := os.LookupEnv("ROOT_DEVICE_PATH")
```

Comment:
What will set this environment variable?

Reply:
VK